packages/a11y/src/index.js

```diff
@@ -0,0 +1,46 @@
+import addContainer from './addContainer';
+import clear from './clear';
+import domReady from '../../domReady/src/';
```

Comment:
I think there's some Lerna magic to leverage here. I think if we add a package to the dependencies of another package, it will automatically be symlinked if the versions match